src/components/SearchBar.jsx

```diff
@@ -21,9 +21,11 @@ export default class SearchBar extends React.Component {
   handleSubmit = async (event) => {
     event.preventDefault();
     const { setCurrentCity } = this.props;
+
     const apiURL = `https://us1.locationiq.com/v1/search?key=${
       import.meta.env.VITE_LOCATIONIQ_API_KEY
     }&q=${this.state.searchTerm}&format=json`;
+
     let currentCity = null;
     try {
```

Comment:
Maybe just a personal style/preference thing, but with REST utilities (i.e. Axios.get()) personally, I like to go with .catch(err => {}) to handle errors rather than a try/catch block, just because I feel like it flows better to my eye. But probably not a big difference either way.

Reply:
Definitely! I actually typically do it that way myself, but haven't used try/catch a whole lot, so I figured I'd play with it and see how I like it.